### PR TITLE
support temporary limits in line creation modification

### PR DESCRIPTION
**Pull request by dbraquart**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/entities/equipment/creation/CurrentTemporaryLimitCreationEmbeddable.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Embeddable
+public class CurrentTemporaryLimitCreationEmbeddable {
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "value_")
```
Comment (Walid-Sahnoun) [inline src/main/java/org/gridsuite/modification/server/entities/equipment/creation/CurrentTemporaryLimitCreationEmbeddable.java:31]: change "value_" to "value"

Comment (laurgarn) [reply src/main/java/org/gridsuite/modification/server/entities/equipment/creation/CurrentTemporaryLimitCreationEmbeddable.java:31]: @Walid-Sahnoun this is on purpose, to circumvent a limitation we encountered in JPA/PostGres.

Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/modification/server/entities/equipment/creation/CurrentTemporaryLimitCreationEmbeddable.java:31]: ok, thanks

File: src/main/java/org/gridsuite/modification/server/dto/CurrentTemporaryLimitCreationInfos.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import lombok.*;
+
+/**
+ * @author David Braquart <david.braquart at rte-france.com>
+ */
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
```
Comment (laurgarn) [inline src/main/java/org/gridsuite/modification/server/dto/CurrentTemporaryLimitCreationInfos.java:20]: Is @Setter necessary ?

**Review by laurgarn — APPROVED** [review]
Tests : OK

File: src/main/java/org/gridsuite/modification/server/dto/CurrentTemporaryLimitCreationInfos.java (hunk 2 above)
Comment (dbraquart) [reply src/main/java/org/gridsuite/modification/server/dto/CurrentTemporaryLimitCreationInfos.java:20]: no it is not. I remove

**Pull request merged**